src/services/config.js

```diff
@@ -156,13 +166,13 @@ class Config {
     if (!fs.existsSync(this.filePath)) {
       return configData;
     }
-
     configData.loadFromObject(await fs.readJSON(this.filePath));
     return configData;
   }
 
   async save(configData) {
     configData = {
+      edge: configData.edge,
       email: configData.email,
       prompts: configData.prompts,
       // Note the historical 'projects' naming.
```

Comment:
Should go ahead and fix the migration comment below.

Reply:
Not sure I understand what this means

Reply:
There's a line further down about Node8 (which is no longer supported).